lib/apr/events.ex

```diff
@@ -40,6 +41,14 @@ defmodule Apr.Events do
       where: e.inserted_at > ago(^value, "day")
   end
 
+
+  def pending_approval_orders do
```

Comment:
this is not used yet, i'll work separately to test this but maybe its ok to have it here